bin/good-first-issue.js

```diff
@@ -34,15 +34,22 @@ cli
       console.log("If you'd like to add a new project to good-first-issue,")
       console.log(
         "Please see the module's Github repository: " +
-          chalk.cyan(
-            'https://github.com/bnb/good-first-issue#adding-new-projects'
-          )
+        chalk.cyan(
+          'https://github.com/bnb/good-first-issue#adding-new-projects'
+        )
       )
       console.log('')
       process.exit(0)
     }
 
-    const issues = await goodFirstIssue(input)
+    var issues
+
+    try {
+      issues = await goodFirstIssue(input)
+    } catch (e) {
+      console.log('Oops! We encountered an issue while searching. Please check your network connection.')
```

Comment:
I think it would be a better approach to actually deal with the exception at this point. It would in this approach return a helpful description with a less pragmatic approach to it, keeping the friendly message but treating the error object as it should would be a better take imo.

Could it be a better idea to actually use this exception to exponentially try to request the resource again, rather than in /all cases return an error whether friendly or not? Think long poll strategy would be less obtrusive